conf/defaults.ini

```diff
@@ -397,6 +397,7 @@ global_session = -1
 [alerting]
 # Makes it possible to turn off alert rule execution.
 execute_alerts = true
+show_alerting_ui = true
```

Comment:
I agree with Torkel with either enable alerting completely or disabling it with a single option. How about changing "execute alerts" to "enable_alerting" and use the same variable for toggling both the alerting in UI and the alert engine in the backend?